distribution/src/main/gateway/bin/scripts/metrics.statsD.start

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I suggest removing the script and the .bat, as suggested in by comment on gateway.start.bat